### PR TITLE
feat: CHK-1538 add configuration for ecommerce PROD env

### PR DESCRIPTION
**Pull request by pietro-tota**
### List of changes
- Add configuration for Prod env, in particular:
  - redis configuration;
  - cosmos configuration;
  - psp list vpos/xpay 
  -  azure storage queue configurations
- Disable alert creation for UAT env since DisableRateLimitingResponses mongo capability cause no 429 response to be sent by Mongo

### Motivation and context

The goal of this PR is to add configuration for ecommerce PROD.

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/ecommerce-common/env/weu-prod/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  capacity = 0
+  sku_name = "Basic"
+  family   = "C"
+}
+
+ecommerce_storage_params = {
+  enabled                       = true
+  tier                          = "Standard"
+  kind                          = "StorageV2"
+  account_replication_type      = "LRS",
```
Comment (infantesimone) [inline src/domains/ecommerce-common/env/weu-prod/terraform.tfvars:80]: [geo-zone-redundant storage](https://learn.microsoft.com/en-us/azure/storage/common/storage-redundancy#geo-zone-redundant-storage)

Comment (infantesimone) [reply src/domains/ecommerce-common/env/weu-prod/terraform.tfvars:80]: GZRS

**Comment by pietro-tota** [comment]
Mark this PR as on hold for now waiting for Redis and storage account prod resource configurations. @infantesimone

File: src/domains/ecommerce-common/env/weu-prod/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  enable_serverless  = false
+  enable_autoscaling = true
+  max_throughput     = 4000
+  throughput         = 1000
+}
+
+redis_ecommerce_params = {
+  capacity = 0
+  sku_name = "Basic"
+  family   = "C"
```
Comment (diegolagospagopa) [inline src/domains/ecommerce-common/env/weu-prod/terraform.tfvars:73]: The Sku for production must be `standard` and please put the version of the redis engine
<https://pagopa.atlassian.net/wiki/spaces/DEVOPS/pages/458654714/Redis+cache#sku-(dimensionamento)>

**Comment by diegolagospagopa** [comment]
Even if is not related to this PR, please review how the UAT env is configured.

Because is possible that in future this env can be used for internal customers, and we need skus more bigger to support so.

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

File: src/domains/ecommerce-common/env/weu-prod/terraform.tfvars (hunk 2 above)
Comment (infantesimone) [reply src/domains/ecommerce-common/env/weu-prod/terraform.tfvars]: I agree, fixed

File: src/domains/ecommerce-common/env/weu-prod/terraform.tfvars [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+#TODO to check subnet
+cidr_subnet_cosmosdb_ecommerce = ["10.1.153.0/24"]
+cidr_subnet_redis_ecommerce    = ["10.1.148.0/24"]
+cidr_subnet_storage_ecommerce  = ["10.1.154.0/24"]
+
+cosmos_mongo_db_ecommerce_params = {
+  enable_serverless  = false
+  enable_autoscaling = true
+  max_throughput     = 4000
```
Comment (infantesimone) [inline src/domains/ecommerce-common/env/weu-prod/terraform.tfvars:66]: 1000 it is a good initial value, increasing as the traffic increases cc @diegolagospagopa @pietro-tota

**Review by diegolagospagopa — APPROVED** [review]

File: src/domains/ecommerce-app/env/weu-prod/terraform.tfvars [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+# PSP for xpay
+# - CIPBITMM  (NEXI)
+ecommerce_xpay_psps_list = "CIPBITMM"
+# PSP for vpos
+# - BNLIITRR (WORLDLINE)
+# - BCITITMM (INTESA)
+# - UNCRITMM (UNICREDIT)
+# - BPPIITRRXXX (POSTE)
+# - CPERRATO (POSTEPAY)
```
Comment (infantesimone) [inline src/domains/ecommerce-app/env/weu-prod/terraform.tfvars:44]: CPERRATO -> PPAYITR1XXX

Comment (pietro-tota) [reply src/domains/ecommerce-app/env/weu-prod/terraform.tfvars]: Fixed with https://github.com/pagopa/pagopa-infra/pull/960/commits/e5376cd85db5db2b46d5a51a9f66f7c7ba151a3e

**Review by infantesimone — APPROVED** [review]

**Pull request merged**